compose.yml

```diff
@@ -0,0 +1,17 @@
+# version: "3"
+
+services:
+  frontend:
+    build: ./frontend/
+    ports:
+      - "80:80"
+
+  backend_rest:
+    build: ./backend/rest/
+    ports:
+      - "8080:8080"
+
+  backend_ws:
+    build: ./backend/ws/
+    ports:
+      - "8030:8030"
```

Comment:
need to take a look at the ports again, I dont think these match the port the ws server is listening at